src/config/env.rs

```diff
@@ -9,6 +9,9 @@ impl ConfigEnv {
         Profile {
             api_key: ConfigEnv::get_override(CT_API_KEY)
                 .or_else(|| ConfigEnv::get_override(CT_OLD_API_KEY)),
+            description: None,
+            environment: ConfigEnv::get_override(CT_ENVIRONMENT),
```

Comment:
I think you could do the `Some`/`None` split here and not make the caller worry about it.